client/js/components/pages/HomePage.js

```diff
@@ -1,10 +1,350 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router';
 
 class HomePage extends Component {
+  constructor () {
+    super();
+    this.state = { visibility: false };
+  }
+
+  onClick () {
```

Comment:
Could you add some more meaningful name? So that it's clear which clicks this callback is for